google-cloud-bigquery/lib/google/cloud/bigquery/load_job.rb

```diff
@@ -222,6 +222,28 @@ def backup?
           val == "DATASTORE_BACKUP"
         end
 
+        ##
+        # Checks if the source format is ORC.
+        #
+        # @return [Boolean] `true` when the source format is `ORC`,
+        #   `false` otherwise.
+        #
+        def orc?
+          val = @gapi.configuration.load.source_format
+          val == "ORC"
```

Comment:
nit: temporary variable probably not necessary

Reply:
Fixed, thanks.